WtWSMS/localization/english/bookmark/bm_476_l_english.yml

```diff
@@ -2,84 +2,84 @@
+ bookmark_476_euric: "Euric"
+ bookmark_476_euric_desc: "The Visigoths have grown from mere #italic foederati#! in 418 to being the most powerful kingdom in what was once the #italic Pars Occidentalis#!. Euric stands atop this kingdom, having killed his own brother to gain the throne and defeating numerous foes to secure his rule. In 475, he forced Julius Nepos to recognize his independence from Rome.\n\n#bold Will Euric lead the Visigoths to greater successes, or will grasping for greater power lead to his downfall?#!"
  
- bookmark_476_gundobad:0 "Gundobad"
- bookmark_476_gundobad_desc:0 "The Burgundians have settled in #italic Sapaudia#! since 443. Gundobad has succeeded his uncle, Ricimer, to rule the Burgundians and is now beginning the process of consolidating his rule. With the deposition of Glycerius, Gundobad's favoured Roman Emperor candidate, in 474, the Burgundians are now #italic de facto#! independent of Rome.\n\n#bold Surrounded by hostile forces, can Gundobad secure the independence of the Burgundians, or will upstarts conquer their kingdom?#!"
+ bookmark_476_gundobad: "Gundobad"
+ bookmark_476_gundobad_desc: "The Burgundians have settled in #italic Sapaudia#! since 443. Gundobad has succeeded his uncle, Ricimer, to rule the Burgundians and is now beginning the process of consolidating his rule. With the deposition of Glycerius, Gundobad's favoured Roman Emperor candidate, in 474, the Burgundians are now #italic de facto#! independent of Rome.\n\n#bold Surrounded by hostile forces, can Gundobad secure the independence of the Burgundians, or will upstarts conquer their kingdom?#!"
  
- bookmark_476_gibuld:0 "Gibuld"
- bookmark_476_gibuld_desc:0 "Gibuld, king of the Alemannians, lies in a precarious position. Childeric of the Salian Franks seems poised to reunite the Merovingians and the Thurginians to the East are becoming hungry for new land. Odovacar seems poised to replace the decadent Western Roman Empire to the South.\n\n#bold Caught between all these foes, can Gibuld resist subjugation at the hands of others, or the Alemannians doomed to be conquered by outsiders?#!"
+ bookmark_476_gibuld: "Gibuld"
+ bookmark_476_gibuld_desc: "Gibuld, king of the Alemannians, lies in a precarious position. Childeric of the Salian Franks seems poised to reunite the Franks and the Thurginians to the East are becoming hungry for new land. Odovacar seems poised to replace the decadent Western Roman Empire to the South.\n\n#bold Caught between all these foes, can Gibuld resist subjugation at the hands of others, or the Alemannians doomed to be conquered by outsiders?#!"
```

Comment:
Typo on Thuringians

Reply:
FYI, feel free to fix typos like that; I can't spell worth anything and everyone should have write access to my PRs.